addons/opsportal/1.1.x/opsportal-12.yaml

```
@@ -0,0 +1,49 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: opsportal
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: opsportal
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-12"
+    appversion.kubeaddons.mesosphere.io/opsportal: "1.0.0"
```

```suggestion
    appversion.kubeaddons.mesosphere.io/opsportal: "1.1.0"
```
@shaneutt @joejulian any idea what this version is for this addon? or any idea who could have an idea? Increasing this now seems to be a bit weird given there is already several versions of konvoy (and kba releases) out with `1.0.0`… shall we align this version with the kba release? is it a new version of `opsportal` at all? 🤔

---

This can be derived from the helm chart:
```
helm search opsportal
NAME                       	CHART VERSION	APP VERSION	DESCRIPTION    
mesosphere-stable/opsportal	0.2.13       	1.0.0      	OpsPortal Chart
```

Technically it looks like we should not make a change here, as the app version does appear to be `1.0.0`.

---

Did this get out of sync in the chart?

---

sorry, true. my question should have been asked in the chart rather than the addon… so consider my question talking about chart 😓

---

So does this need to change for this PR?

---

If you're not in a terrible rush to get this update out, then I would say let's fix it here.
If you are in a terrible rush, let's make a JIRA to make sure this gets patched next time we comb over this addon.

---

:point_up: